meta-openpli/recipes-multimedia/tuxtxt/libtuxtxt.bb

```diff
@@ -7,6 +7,7 @@ inherit gitpkgv
 
 GITHUB_URI ?= "git://github.com"
 SRC_URI = "${GITHUB_URI}/OpenPLi/tuxtxt.git"
+SRC_URI_append_libc-musl = " file://0001-libtuxtxt-fix-build-with-musl.patch"
```

Comment:
@andrea-adami Would you please send a PR for https://github.com/OpenPLi/tuxtxt or @WanWizard  could apply this patch.